pages/404.page.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import Link from "next/link";
+import Image from "next/image";
+import styles from "../styles/404.module.css";
+import dynamic from "next/dynamic";
+const Footer = dynamic(() => import("../components/Footer"));
```

Comment:
invalid path!

Reply:
> invalid footer path!

It's valid. or don't want footer in error page ?